tutorials/logical_audio_sensor.md

```diff
@@ -0,0 +1,195 @@
+
+```
+/model/blue_box/sensor/source_1/play
+/model/blue_box/sensor/source_1/stop
+/model/red_box/sensor/source_1/play
+/model/red_box/sensor/source_1/stop
+```
+
+Let's start the source attached to `blue_box`.
+This can be done by running the following command, which calls the "play" service for the source attached to `blue_box`:
+
+```
+ign service -s /model/blue_box/sensor/source_1/play --reqtype ignition.msgs.Empty --reptype ignition.msgs.Boolean --timeout 1000 --req 'unused: false'
+```
+
+Now, if you look back at the terminal that is displaying the output of `green_box`'s microphone detections, we can see that this microphone is detecting audio from `blue_box`'s source (we see `blue_box` as a part of the `key` field):
+
+```
+header {
+  stamp {
+    sec: 255
+    nsec: 673000000
+  }
+  data {
+    key: "world/logical_audio_sensor/model/blue_box/sensor/source_1"
+  }
+}
+data: 0.6
+```
+
+You can also echo `yellow_box`'s microphone detection topic, and you should see that this microphone can detect audio from `blue_box`'s source as well.
+
+Since the source attached to `blue_box` was configured with `<play_duration>0</play_duration>` in SDF, it won't stop playing unless we call the stop service on it.
+Before we call the stop service on this source, move `blue_box` around with the transform control tool - you'll see that once `blue_box`'s source volume falls below a microphone's detection threshold (defined in the SDF via `<volume_threshold>`), messages will stop being published to the microphone's detection topic.
+
+Let's go ahead and call the stop service to make sure that this source will stop playing.
+Move `blue_box` back towards its original position, until you see detection messages being published by the microphones attached to `green_box` and/or `yellow_box`.
+Now, go ahead and stop `blue_box`'s source by running the following command:
+
+```
+ign service -s /model/blue_box/sensor/source_1/stop --reqtype ignition.msgs.Empty --reptype ignition.msgs.Boolean --timeout 1000 --req 'unused: false'
+```
+
+Now, if you look at the output for either microphone topic, you'll notice that no new messages are being published, which makes sense since no audio sources are currently playing.
+You can also try to call the play/stop services on the source attached to the `red_box`.
+Since the `red_box` has a play duration of 10 seconds, the source will automatically stop playing 10 seconds (of sim time) after starting it, unless you stop it earlier by calling the stop service.
+One other important thing to note is that calling the play service on a source that is already playing does nothing - so, if the source attached to `red_box` is already playing, and you call the play service on it, it will not play for another 10 seconds - the source will still stop at the original 10 second duration mark.
```

Comment:
```suggestion
One other important thing to note is that calling the play service on a source that is already playing does nothing. For example, if the source attached to `red_box` is already playing and you call the play service on it, then it will not play for another 10 seconds. The source will still stop at the original 10 second duration mark.
```

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/401/commits/bb50ab5e21d6fec3c54a1e1b7fdd6623fc285908